migration_original/ODS1Stage/tables/Base/FacilityHours/spu_translated_FacilityHours.sql

```diff
@@ -11,6 +11,7 @@ DECLARE
 -- Base.FacilityHours depends on: 
 --- FACILITY_PROFILE_PROCESSING
 --- Base.Facility
+--- Base.DaysOfWeek
```

Comment:
Add Raw.VW_FACILITY_PROFILE here

Reply:
Since the view is created by us and not an ODS1stage original object I was writing FACILITY_PROFILE_PROCESSING instead of the view